omero/sysadmins/unix/install-web/install-nginx.txt

```diff
@@ -161,92 +160,55 @@ Benchmark
 We run example benchmarks on rendering thumbnails and 512x512 pixels planes
 for 100 concurrent users making 5000 requests in total::
 
-    $ ab -n 5000 -c 100 https://server.openmicroscopy.org/omero/webclient/render_thumbnail/size/96/1234/
-    This is ApacheBench, Version 2.3 <$Revision: 1430300 $>
+    $ ab -n 5000 -c 100 https://server.openmicroscopy.org/omero/webclient/render_thumbnail/1234/
+    This is ApacheBench, Version 2.3 <$Revision: 1706008 $>
```

Comment:
this is different
This is used for measurement performance of web servers
The web server does not have to be Apache.